coremltools/test/test_shaper_bindings.py

```diff
@@ -61,7 +61,7 @@ def setUpClass(self):
                 layer.convolution.kernelChannels = channels
                 layer.convolution.kernelSize.append(1)
                 layer.convolution.kernelSize.append(1)
-                layer.convolution.same.MergeFromString('')
+                layer.convolution.same.MergeFromString(b'')
```

Comment:
this doesn't cause any issues with python 2?

Reply:
No, it's fine. We're doing this same things in several other places in the code base. In python 2.7 `b''` just returns a `str`.